kinder/pkg/cluster/manager/actions/actions.go

```diff
@@ -238,6 +246,25 @@ func ValidateCopyCertsMode(t CopyCertsMode) error {
 	return nil
 }
 
+// SupportedKubeadmConfigVersion returns the list of supported kubeadm config version
+func SupportedKubeadmConfigVersion() []string {
+	return []string{
+		"v1beta1",
+		"v1beta2",
+	}
+}
+
+// ValidateKubeadmConfigVersion validates a kubeadm config version
+func ValidateKubeadmConfigVersion(v string) error {
```

Comment:
This is a little bit trickier than this, because the kubeadm config version is linked to the kubeadm version we are using. Not sure we can really catch this during flag validation, but let's make sure to keep this in my down the path

Reply:
Emm... Maybe it's better to validate the config version through `Kubeadm` rather than `Kinder`?

Reply:
Kinder can "hardcode" this, it requires the same info you already used in `GetKubeadmConfigVersion` but in this case you have to check if the version provided by the user is supported by the Kubernetes version.

However, I'm fine also if we didn't validate consistence of config version vs kubeadm version  and just assume that a user playing with kinder knows which version to use.